operations/poplar.go

```diff
@@ -147,10 +153,30 @@ func poplarReport() cli.Command {
 				Name:  apiKeyHeaderFlagName,
 				Usage: "specify the API key header for API authentication",
```

Comment:
Not necessary, but if you want to clean these other flag messages up that would be great! We can remove the `specify the` prefix since it is redundant.